sdk/core/azure-core/src/main/java/com/azure/core/util/MetricsOptions.java

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.util;
+
+import com.azure.core.http.ProxyOptions;
+
+/**
+ * Metrics configuration options for clients.
+ */
+public class MetricsOptions {
+    private static final ConfigurationProperty<Boolean> IS_DISABLED_PROPERTY = ConfigurationPropertyBuilder.ofBoolean("metrics.disabled")
+        .environmentVariableName(Configuration.PROPERTY_AZURE_METRICS_DISABLED)
+        .shared(true)
+        .defaultValue(false)
+        .build();
+
+    private boolean isEnabled;
+
+    /**
+     * Creates new instance of {@link MetricsOptions}
+     */
+    public MetricsOptions() {
+        isEnabled = !Configuration.getGlobalConfiguration().get(IS_DISABLED_PROPERTY);
+    }
+    /**
+     * Attempts to load metrics options from the configuration.
+     *
+     * {@code null} will be returned if no metric options are found in the environment.
+     *
+     * @param configuration The {@link Configuration} that is used to load proxy configurations from the environment. If
+     * {@code null} is passed then {@link Configuration#getGlobalConfiguration()} will be used.
+     * @return A {@link ProxyOptions} reflecting a proxy loaded from the environment, if no proxy is found {@code null}
+     * will be returned.
+     */
+    public static MetricsOptions fromConfiguration(Configuration configuration) {
+        if (configuration.contains(IS_DISABLED_PROPERTY)) {
+            return new MetricsOptions().setEnabled(!configuration.get(IS_DISABLED_PROPERTY));
+        }
+
+        return null;
```

Comment:
Should we return an empty `MetricsOptions` instance instead of a `null`? For this particular type, there's just one property now, so, there might not be any further chaining necessary. But, as a general pattern, it's good to return an instance that's configured with whatever properties were available in `Configuration` (empty if nothing's set) and then the user can continue chaining to set other properties.
```java
FooOptions options = FooOptions.fromConfiguration(config) // returns empty instance if no properties were found in config
                    .setFoo(foo)
                    .setBar(bar) ;
```